.expeditor/scripts/release_habitat/package_and_upload_binary.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
+# Unpack the hart file from our channel, repack it, and upload it to
+# package-router
+
+set -euo pipefail
+
+source .expeditor/scripts/release_habitat/shared.sh
+
+export HAB_AUTH_TOKEN="${ACCEPTANCE_HAB_AUTH_TOKEN}"
+export HAB_BLDR_URL="${ACCEPTANCE_HAB_BLDR_URL}"
+
+channel=$(get_release_channel)
+
+echo "--- Channel: $channel - bldr url: $HAB_BLDR_URL"
+
+release_version=$(get_latest_pkg_release_version_in_release_channel "hab")
+
+declare -g hab_binary
+install_release_channel_hab_binary "x86_64-linux"
+
+${hab_binary} pkg install core/gzip
+${hab_binary} pkg install core/tar
+${hab_binary} pkg install core/wget
+${hab_binary} pkg install core/zip
+
+# Import packages@chef.io GPG signing key
+aws s3 cp s3://chef-cd-citadel/packages_at_chef.io.pgp packages_at_chef.io.pgp --profile=chef-cd
+gpg --import packages_at_chef.io.pgp
+
+#TODO make this nicer
```

Comment:
Remove the TODO unless you plan to update this in another PR in the near future.

Reply:
Good eye - that was before I finished :)